package.json

```diff
@@ -23,11 +23,12 @@
         "dev": "yarn cli prepare-link && yarn watch:all; yarn cli prepare-link --revert",
         "format": "yarn lint --fix",
         "lint": "eslint ./packages/**/*.{ts,js} --quiet",
-        "loop": "yarn workspaces foreach -Apti --include \"@datadog/*\" --exclude \"@datadog/build-plugins\"",
+        "loop-published": "yarn workspaces foreach -A --include \"@datadog/*\" --exclude \"@datadog/build-plugins\"",
```

Comment:
you might want to use the `-t` flag (topological) to publish dependencies first, in case something goes wrong

Reply:
I'm not sure it's really necessary.

It's not publishing anything, it's a simple loop helper over the workspaces that we publish (eg. `@datadog/*-plugin`).

I wanted (needed) a flat, sequential loop in order to (in this case) bump the versions in their `package.json`.

For the needs of topological, I'd use the `loop` helper, that adds the `-pti` flags to it (to also make it parallel).